automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -142,6 +142,16 @@ trait Orchestration extends FireCloudClient with LazyLogging {
     postRequestWithMultipart(Config.FireCloud.orchApiUrl + "api/workspaces/" + ns + "/" + wsName + "/" + "importEntities", fileName, fileContent)
   }
 
+  def importMethodConfigFromRepo(ns: String, wsName: String, configurationNamespace: String, configurationName: String, configurationSnapshotId: Int, destinationNamespace: String, destinationName: String)(implicit token: AuthToken) = {
```

Comment:
Also, if it's in a `methodConfigs` object context, the function name can be shortened to `importFromMethodRepo` (or `copyFromMethodRepo` to make it more closely match the API vocabulary).